nengo/spa/actions.py

```diff
@@ -165,37 +164,28 @@ def __getitem__(self, key):
             return Module(key)
 
 
-class Actions(Config):
+class Actions(object):
     """A collection of Action objects.
 
     The *args and **kwargs are treated as unnamed and named Actions,
-    respectively.  The list of actions are only generated once process()
-    is called, since it needs access to the list of module inputs and
-    outputs from the SPA object. The **kwargs are sorted alphabetically before
-    being processed.
+    respectively.
 
     The keyword argument `vocabs` is special in that it provides a dictionary
     mapping names to vocabularies. The vocabularies can then be used with those
     names in the action rules.
     """
 
-    def __init__(self, *args, **kwargs):
-        super(Actions, self).__init__(BasalGanglia, Thalamus)
+    def __init__(self, *args, vocabs=None, **kwargs):
```

Comment:
Python2 doesn't support this syntax.  :(  Here's an alternate implementation that I think does what you're intending here:

```python
   def __init__(self, *args, **kwargs):
        super(Actions, self).__init__()

        self.actions = []
        self.named_actions = {}

        self.construction_context = None

        self.parse(*args, **kwargs)

    def parse(self, *args, **kwargs):
        if kwargs.has_key('vocabs'):
            vocabs = kwargs['vocabs']
            del kwargs['vocabs']
        else:
            vocabs = None

        sorted_kwargs = sorted(kwargs.items())
        ...
```